paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetResult.kt

```diff
@@ -1,12 +1,17 @@
-package com.stripe.android.paymentsheet.wallet
+package com.stripe.android.customersheet
 
+import android.content.Intent
+import android.os.Bundle
 import androidx.annotation.RestrictTo
+import androidx.core.os.bundleOf
 import com.stripe.android.ExperimentalCustomerSheetApi
 import com.stripe.android.paymentsheet.model.PaymentOption
+import com.stripe.android.view.ActivityStarter
+import java.io.Serializable
 
 @ExperimentalCustomerSheetApi
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-sealed class CustomerSheetResult {
+sealed class CustomerSheetResult : Serializable {
```

Comment:
Sorry. I should have been more clear the first time.

I think we can do this without requiring any serialization/parcelization. 

Maybe we can pair on this to get it working.